packages/office-ui-fabric-react/src/components/Slider/examples/Slider.Basic.Example.tsx

```diff
@@ -24,6 +24,16 @@ export class SliderBasicExample extends React.Component<{}, ISliderBasicExampleS
           showValue={true}
           onChange={(value: number) => console.log(value)}
         />
+        <Slider
+          label="Snapping slider example"
+          min={1}
+          max={5}
+          step={1}
+          defaultValue={2}
+          showValue={true}
+          onChange={(value: number) => console.log(value)}
+          snaps={true}
```

Comment:
snaps does not need to be set to true. Just like `disabled` or `checked`, properties are assumed true unless you specify false like `snaps={false}`

Reply:
Hmmm if this is true the other examples would all be snapping though by default and they're not...

Reply:
adding the 'snaps' prop is all you need to do to set it to true. snaps={true} is redundant. I know my comment was a bit confusing

Reply:
Ohh I see what you're saying now. @Sneha (not sure why it's not letting me tag you), you should be able to just do snaps kind of like the originFromZero boolean prop in the last Slider example or disabled here: https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/experiments/src/components/Button/examples/Button.Example.tsx#L37. It's probably best to change disabled in the Slider example below to match this too.